recipe/meta.yaml

```diff
@@ -16,21 +16,24 @@ build:
 
 requirements:
   host:
-    - python >=3.9,<4.dev0
-    - flit-core >=3.2,<4
+    - python {{ python_min }}
+    - flit-core ==3.10.1
     - pip
   run:
-    - python >=3.9,<4.dev0
+    - python >={{ python_min }}
     - apache-airflow >=2.8.0
     - apache-beam >=2.53.0
     - pyarrow >=14.0.1
+    # temporary constraint to get past pip check
+    - async-timeout >=4.0,<5.0
```

Comment:
Needed because aiohttp 3.10.10 was missing this constraint in build 0, fixed in:
https://github.com/conda-forge/aiohttp-feedstock/pull/107